package.json

```diff
@@ -1,7 +1,19 @@
 {
-  "name": "node-irrelon-router",
-  "version": "0.1.0",
+  "name": "node-akxe-router",
```

Comment:
Should not change the name of the project unless you want to deploy this yourself to NPM.

Reply:
Sorry my bad